### PR TITLE
Add more granularity

### PR DESCRIPTION
**Pull request by herulume**

### CODE REVIEW

**Review by miguelbrandao — APPROVED** [review]

File: LipParse/app/Main.hs [hunk 1]
```diff
@@ -1,13 +1,18 @@
 module Main where
 
-import Parse 
+import           Parse
```
Comment (ZePedroResende) [inline LipParse/app/Main.hs:3]: is this right ?

File: LipParse/src/Parse.hs [hunk 2]
```diff
@@ -44,15 +26,19 @@ changeHeaders path content opath = do
 
 getData :: FilePath -> IO (Vector Person)
 getData path  = do
-    cdvData <- BL.readFile path 
+    cdvData <- BL.readFile path
     case decodeByName cdvData of
-        Left r -> error r
+        Left r      -> error r
```
Comment (ZePedroResende) [inline LipParse/src/Parse.hs]: :thinking:

File: LipParse/app/Main.hs (hunk 1 above)
Comment (herulume) [reply LipParse/app/Main.hs:3]: Yes, default in haskell due to qualified imports

File: LipParse/src/Parse.hs (hunk 2 above)
Comment (herulume) [reply LipParse/src/Parse.hs]: Same

**Pull request merged**